mycore-mods/src/main/java/org/mycore/mods/merger/MCRAbstractRedundantModsEventHandler.java

```diff
@@ -0,0 +1,139 @@
+package org.mycore.mods.merger;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jdom2.Element;
+import org.mycore.common.MCRConstants;
+import org.mycore.common.events.MCREvent;
+import org.mycore.common.events.MCREventHandlerBase;
+import org.mycore.datamodel.metadata.MCRObject;
+import org.mycore.mods.MCRMODSSorter;
+import org.mycore.mods.MCRMODSWrapper;
+import org.mycore.mods.classification.MCRClassMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Checks for and removes redundant classifications in Mods-Documents. If a classification category and
+ * the classification's child category are both present in the document, the parent classification will
+ * be removed. The processed document will be finally be sorted using {@link MCRMODSSorter}.<br><br>
```

Comment:
Every handler would end up sorting again, which is not necessary. I would keep the operation more atomic, possibly in a separate event handler.

Reply:
Do you mean creating yet another EventHandler that does nothing but sorting the document?

Reply:
In principle, merging, sorting and so on are post processing for me. I'm not sure whether it generally makes sense to always write a new event handler for this or whether it would make more sense to write an event handler for which you can configure all post process actions.

Reply:
I'll make a note to discuss this at the next developer meeting. But I don't see this in the scope of this ticket, so I'll resolve the discussion.